<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dương Thanh Bình K244060766</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>LearnQListWidgetEmployee/Images/K244060766.jpg</normaloff>LearnQListWidgetEmployee/Images/K244060766.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Employee Management</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>591</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="title">
     <string>Employee Information:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButonWoman">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Woman</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditname">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMan">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Man</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>591</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="title">
     <string>Action:</string>
    </property>
    <widget class="QPushButton" name="pushButton_new">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>LearnQListWidgetEmployee/Images/add.ico</normaloff>LearnQListWidgetEmployee/Images/add.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>LearnQListWidgetEmployee/Images/save.ico</normaloff>LearnQListWidgetEmployee/Images/save.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>LearnQListWidgetEmployee/Images/remove.ico</normaloff>LearnQListWidgetEmployee/Images/remove.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>LearnQListWidgetEmployee/Images/delete.ico</normaloff>LearnQListWidgetEmployee/Images/delete.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>591</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="title">
     <string>List of Employee:</string>
    </property>
    <widget class="QListWidget" name="listWidgetEmployee">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>551</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
